frontend/src/index.js

```diff
@@ -17,15 +17,16 @@ const rootElement = document.getElementById("root");
 render(
   <ChakraProvider>
     <BrowserRouter>
+      <Navbar />
       <Routes>
-        <Route path="/" element={<Navbar />} />
+        <Route path="/" element={<HomePage />} />
```

Comment:
The `/home` path no longer links to the home page. I think we'll have to decide on either changing `/home` to `/` or just starting the app on `/home`.